### PR TITLE
fix: also ignore Num_Lock state

### PR DESCRIPTION
**Pull request by igolskyi**
This is a fix for https://github.com/pbxqdown/gnome-shell-extension-transparent-window/pull/19 because it was not done.

### CODE REVIEW

**Comment by igolskyi** [comment]
@pbxqdown as agreed, keep the update for **also ignore Num_Lock state** https://github.com/pbxqdown/gnome-shell-extension-transparent-window/pull/19

File: extension.js [hunk 1]
```diff
@@ -128,7 +129,7 @@ function destroyOverlay() {
 function onHotkeyPressed() {
   Log.debug("Hot key pressed");
   //Clear the lock bit so the status of Caps_Lock won't affect the functionality
```
Comment (pbxqdown) [inline extension.js:131]: Could you move this comment to utils.js so we know what ~2(caps_lock) and ~16(num_lock) are.

**Comment by pbxqdown** [comment]
@igolskyi Thanks for this!

**Comment by igolskyi** [comment]
@pbxqdown please check updated comments.

**Comment by pbxqdown** [comment]
Looks good! Merging

**Pull request merged**